Flow.Launcher.Plugin/Result.cs

```diff
@@ -83,9 +93,9 @@ public string IcoPath
         public IList<int> SubTitleHighlightData { get; set; }
 
         /// <summary>
-        /// Only results that originQuery match with current query will be displayed in the panel
+        /// Query information associated with the result
         /// </summary>
-        internal Query OriginQuery { get; set; }
+        public Query OriginQuery { get; set; }
```

Comment:
Any practical reason for making this public?

Reply:
We stop using this to filter result, so no reason to mark it as internal. Either we can remove it to avoid some duplicate assignments or just mark it as public so plugin user may access it.

Reply:
if we can remove it, I vote that we do.. seems like it can prove confusing...

Reply:
lets revert back to internal and remove later

Reply:
doing this now